locales/en/allocations.json

```diff
@@ -24,5 +24,36 @@
   },
   "allocation_criteria": {
     "page_title": "Allocation criteria"
+  },
+  "view": {
+    "page_title": "View allocation",
+    "heading": "{{count}} person from {{from_location}} to {{to_location}}, {{date}}",
+    "heading_plural": "{{count}} people from {{from_location}} to {{to_location}}, {{date}}",
+    "criteria_for_allocation": "Criteria for allocation",
+    "created_at": "Created {{datetime}}",
+    "fields": {
+      "estate_type": "Estate type",
+      "category": "Category",
+      "complex_cases": "Complex cases for prisons to agree",
+      "none_provided": "None provided",
+      "other_criteria": "Other criteria",
+      "sentence_length": {
+        "label": "Time left to serve"
+      }
+    },
+    "complete_in_full": "This allocation should be filled in full - consider prisoners that are the next best for this allocation",
+    "people": {
+      "slots_filled": "{{slots_filled}} of {{total_slots}} {{person_or_people}} allocated",
+      "heading": "People for allocation",
+      "empty_slot": "People to be allocated"
+    },
+    "cancel_allocation": "Cancel allocation",
+    "allocation_details": "Allocation details",
+    "sidebar": {
+      "number_of_prisoners": "Number of people",
+      "move_from": "Move from",
```

Comment:
This content already exists within the existing move journeys. We should try and re-use those if possible to keep consistency between the journeys.

Reply:
So, in the past, when I recycled labels you told me not to because the copy might diverge over time. Do we recycle or not?

Reply:
That would have been a different case. In this instance the label is being used to refer to the same thing as the other dashboard. And in those cases we should remain consistent.